photon-core/src/main/java/org/photonvision/vision/camera/USBCameras/GenericUSBCameraSettables.java

```diff
@@ -95,9 +95,11 @@ protected void setUpExposureProperties() {
         // first.
         var autoExpProp = findProperty("exposure_auto", "auto_exposure");
 
-        exposureAbsProp = expProp.get();
-        this.minExposure = exposureAbsProp.getMin();
-        this.maxExposure = exposureAbsProp.getMax();
+        if (expProp.isPresent()) {
+            exposureAbsProp = expProp.get();
+            this.minExposure = exposureAbsProp.getMin();
+            this.maxExposure = exposureAbsProp.getMax();
+        }
```

Comment:
Lol this is the same diff as https://github.com/PhotonVision/photonvision/pull/1556/files#diff-834c9690f133fe3705b29af038bd5e7ca77926e81b5d95123a4768416210a865 . You may also need to at this on line 189 as well
```
 if (autoExposureProp != null) autoExposureProp.set(PROP_AUTO_EXPOSURE_ENABLED);
```

Reply:
Makes sense, I didn't get to that line since it doesn't have the property to begin with. Good to check either way, adding.

Reply:
Fixed. Not sure the timing of that other PR, but this unblocks me, so they can rebase 🙂

Reply:
yep out of scope of that PR anyways :))))))))))))))